src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
maybe we can use the releases/pull request API to generate this and read it from github?